### DISCUSSION TITLE
Deprecation of Ethereum Testnets in Torus Wallet

### DISCUSSION THREAD

**Discussion by yashovardhan** [Web3Auth/Web3Auth#908] [Announcements]
Hey Everyone

Starting this week, ie. by 25th November 2022,  we will be **removing Ropsten, Rinkeby & Kovan Testnets from Torus Wallet's default network list**. This move has been done because of the network deprecation by our node providers and the larger Ethereum community. If you intend to use a testnet going forward, use Goerli. We still support the addition of custom networks, so if you still wish to use any of these networks, you can follow that route.

Feel free to ask us any further questions in this thread.

**Comment by atifnimran** [comment]
Right now
